<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="t_result">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
qproperty-alignment:'AlignVCenter | AlignRight';
border-bottom: 1px solid gray;
} 
background-color:white;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_AC">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_pm">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_percent">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_del">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_pr">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_min">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_plus">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_p">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>151</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_eq">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
